src/components/radios/index.md.njk

```diff
@@ -19,19 +21,19 @@ Don’t use the Radios component if users might need to select more than one opt
 
 ## How it works
```

Comment:
Not part of this PR but might be worth fixing here anyway: there's a mistake closing a link on line 20 of this file `[Checkboxes component](/components/checkboxes/] `

Reply:
Fixed